server/planning/feed_parsers/onclusive.py

```diff
@@ -87,25 +92,55 @@ def parse_item_meta(self, event, item):
         item["definition_short"] = (
             event["description"] if (event["summary"] != "" and event["summary"] is not None) else ""
         )
-        item["links"] = [event["website1"]]
+
+        item["links"] = [event[key] for key in ("website", "website2") if event.get(key)]
 
     def parse_event_details(self, event, item):
-        start_date = self.datetime(event["startDate"])
-        end_date = self.datetime(event["endDate"])
-        timezone = None
-        if event.get("timezone"):
-            timezone = event["timezone"]["timezoneAbbreviation"]
+        if event.get("time"):
+            start_date = self.datetime(event["startDate"], event.get("time"), event["timezone"])
+            end_date = self.datetime(event["endDate"], "23:59:59", event["timezone"])
+            tz = self.parse_timezone(start_date, event)
+            item["dates"] = dict(
+                start=start_date,
+                end=end_date,
+                tz=tz,
+                no_end_time=True,
+            )
+        else:
+            item["dates"] = dict(
+                start=self.datetime(event["startDate"], "00:00:00"),
```

Comment:
Are times **never** included in the `startDate` and/or `endDate` attributes?

Reply:
it's always `00:00:00`